frontend/lib/package.json

```diff
@@ -38,9 +38,9 @@
     "@emotion/is-prop-valid": "^1.2.0",
     "@emotion/react": "^11.10.5",
     "@emotion/styled": "^11.10.5",
-    "@glideapps/glide-data-grid": "6.0.4-alpha3",
-    "@glideapps/glide-data-grid-cells": "6.0.4-alpha3",
-    "@glideapps/glide-data-grid-source": "6.0.4-alpha3",
+    "@glideapps/glide-data-grid": "6.0.4-alpha7",
+    "@glideapps/glide-data-grid-cells": "6.0.4-alpha7",
+    "@glideapps/glide-data-grid-source": "6.0.4-alpha7",
```

Comment:
just out of curiosity: what is the release strategy of glide and is it safe to use alpha versions? Since you are a glide maintainer I assume yes 🙂

Reply:
Yep, we have been running on "alpha" releases for quite some time since we need some of the features & fixes in the latest state. But these alpha releases are pretty stable.